### PR TITLE
implemented getCategoryTranslation, getSectionTranslation and getArticleTranslation.

### PR DESCRIPTION
**Pull request by Walti91**
added methods to fetch a single translation for a category, section or an article. as seen here https://developer.zendesk.com/api-reference/help_center/help-center-api/translations/#show-translation

### CODE REVIEW

File: src/main/java/org/zendesk/client/v2/Zendesk.java [hunk 1]
```diff
@@ -2333,6 +2340,14 @@ public Iterable<Translation> getCategoryTranslations(Long categoryId) {
                 tmpl("/help_center/categories/{categoryId}/translations.json").set("categoryId", categoryId),
                 handleList(Translation.class, "translations"));
     }
+
+    public Translation getCategoryTranslation(long categoryId, String locale) {
```
Comment (PierreBtz) [inline src/main/java/org/zendesk/client/v2/Zendesk.java:2344]: I don't see any endpoint to show translations for Category, am I missing something?

File: src/main/java/org/zendesk/client/v2/Zendesk.java [hunk 2]
```diff
@@ -2242,6 +2242,13 @@ public Iterable<Translation> getArticleTranslations(Long articleId) {
                 handleList(Translation.class, "translations"));
     }
 
+    public Translation getArticleTranslation(long articleId, String locale) {
```
Comment (PierreBtz) [inline src/main/java/org/zendesk/client/v2/Zendesk.java:2245]: Shouldn't we use the same verb as the Zendesk API: https://developer.zendesk.com/api-reference/help_center/help-center-api/translations/#show-translation:

```suggestion
    public Translation showArticleTranslation(long articleId, String locale) {
```

File: src/main/java/org/zendesk/client/v2/Zendesk.java [hunk 3]
```diff
@@ -2384,6 +2399,14 @@ public Iterable<Translation> getSectionTranslations(Long sectionId) {
                 tmpl("/help_center/sections/{sectionId}/translations.json").set("sectionId", sectionId),
                 handleList(Translation.class, "translations"));
     }
+
+    public Translation getSectionTranslation(long sectionId, String locale) {
```
Comment (PierreBtz) [inline src/main/java/org/zendesk/client/v2/Zendesk.java:2403]: I don't see any endpoint to show translations for Section either.

**Review by PierreBtz — CHANGES_REQUESTED** [review]
Thanks for the contribution!

We need to sort out if the endpoint you are using for category and section exist.

We will also need to add some tests to cover the new method(s). I know that it's a bit hard to write those tests since external contributors don't have access to our Zendesk sandbox. In case you are not able to write those, I can do it for you when I have some time.

**Comment by Walti91** [comment]
Hi!

I renamed the methods to match the API documentation which is indeed missing the endpoints for categories and sections, but they do exist and are working. I also added tests for the new methods.

Kind regards

**Comment by PierreBtz** [comment]
@Walti91 I opened a support ticket with Zendesk to understand why those APIs are not documented, we cannot really rely on them if the lack of documentation is on purpose and they are internal APIs for some reason. I'll keep you updated.

**Review by PierreBtz — APPROVED** [review]
Zendesk support confirmed there is a documentation issue on their end, so I think we can move in with this PR.
Thanks for the contribution.

**Comment by PierreBtz** [comment]
Released: https://github.com/cloudbees-oss/zendesk-java-client/discussions/547

**Pull request merged**